.gitlab-ci.yml

```diff
@@ -71,15 +78,36 @@ check-line-width:
   allow_failure:                   false
 
 
-test-linux-stable:                 &test
+cargo-audit:
+  stage:                           test
+  <<:                              *compiler_info
+  only:                            *not-prs
+  script:
+    - cargo audit
+  allow_failure:                   true
+  tags:
+    - linux-docker
+
+
+test-build-subkey:
+  stage:                           test
+  <<:                              *compiler_info
+  only:                            *not-prs
```

Comment:
maybe one could make use of the gitlab [only on changes](https://docs.gitlab.com/ee/ci/yaml/#onlychangesexceptchanges) feature here. or does subkey depend on the source of the rest?

Reply:
It was requested in https://github.com/paritytech/substrate/issues/2735 to be tested in CI. At least on master and v1.0 branches.

Reply:
This still needs to be run for every pr. To make sure it compiles.

Reply:
but could it be limited to run only for changes in the `subkey` directory or can changes in the rest of the repo also break it?

Reply:
Changes in the rest can also break it.

Reply:
@bkchr you mean to set it for every commit in PRs? It takes quite a time to be built.

Reply:
But isn't that run in parallel to the other pipelines?

Reply:
yes, it runs in parallel with other jobs. 
I want to be sure it's that important that we really want to run it every commit.

Reply:
Are the jobs all running on the same machine?
Nevertheless, we need to know when a build breaks something. Having jobs running on master without reporting, does not help anyone either.

Reply:
replaced it with `cargo check --release` should serve the purpose and do it faster.
https://gitlab.parity.io/parity/substrate/-/jobs/159162 - 1'12, not that bad, can afford running on PRs.